docs/Node Operators/running-a-chainlink-node.md

```diff
@@ -16,10 +16,41 @@ It's important to note that nodes can fulfill requests for open APIs out-of-the-
 
 If you would like to provide data from an authenticated API, you can add an [external adapter](../external-adapters/) to enable connectivity through the Chainlink node.
 
-Hardware requirements are light. The only heavy part is you'll need a blockchain node connection. If you use a 3rd party (defined below), you can use a machine with as little as 10GB of storage and 4GB of RAM.
-
 ![Chainlink Node Diagram](/files/ab5762f-end-to-end-diagram.png)
 
+# Hardware Requirements
+
+## Chainlink Node
+
+Your Chainlink node should be run on a server that has a public IP address.
+
+### Minimum
+
+To get started running a Chainlink node, you will need a machine with at least **2 cores** and **4 GB of RAM**. 
```

Comment:
```suggestion
**2 cores**
```
Do we want to provide a cutoff for specific processor generations? For example, on the extreme end I wouldn't expect a [Core 2 Duo E6300](https://ark.intel.com/content/www/us/en/ark/products/27248/intel-core2-duo-processor-e6300-2m-cache-1-86-ghz-1066-mhz-fsb.html) would be sufficient.

I assume we don't support ARM either. Correct?

Reply:
Just checked with the team and we should be okay without specifying the additional details mentioned.
With regards to the first point, we don't want to be too specific since we will need to keep updating if so.
With regards to the second point, as far as we know, it is supported, though we may need to do some testing in the future.